src/GUI_draft/templates/landing_page.html

```diff
@@ -49,4 +48,10 @@ <h4><a href="/project_list">Project List VC</a></h4>
 
 </div>
 
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```

Comment:
This does not need to be imported twice, it is already in `base.html`

Reply:
Removed now